src/engine-actions/is-node-config-valid.js

```diff
@@ -24,7 +33,7 @@ async function validateNodeConfig () {
     throw new Error(`Mismatched configuration: Engine is configured for ${this.currencyConfig.chainName}, LND is configured for ${chainName}.`)
   }
 
-  this.validated = true
+  return true
```

Comment:
Removed assignment of `this.validated` to the LndEngine constructor for readability, instead of setting different engine properties nested in other functions